js/main.js

```diff
@@ -19,6 +19,8 @@
   var NAMES = ['Артем', 'Вася', 'Матрена', 'Катюша', 'Слава', 'Поля'];
   var AVATARS = ['1', '2', '3', '4', '5', '6'];
   var PHOTO_ITEMS_COUNT = 25;
+  var ENTER_KEYCODE = 13;
```

Comment:
```suggestion
  var ENTER_KEY_CODE = 13;
```
нет такого английского слова keycode. это два слова и они пишутся раздельно